index.html

```diff
@@ -0,0 +1,127 @@
+          <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png" alt="project">
+          <p class="project-title">
+           
+            Random Quote Machine
+            
+          </p>
+        </a>
+        <a href="https://codepen.io/freeCodeCamp/full/wgGVVX" target="_blank" class="project project-tile">
+          <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt="project">
+          <p class="project-title">
+           
+            JavaScript Calculator
+           
+          </p>
+        </a>
+        <a href="https://codepen.io/freeCodeCamp/full/mVEJag" target="_blank" class="project project-tile">
+          <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt="project">
+         
+            <p class="project-title">
+              Map Data Across the Globe
+            </p>
+            
+        </a>
+        <a href="https://codepen.io/freeCodeCamp/full/wGqEga" target="_blank" class="project project-tile">
+          <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png" alt="project">
+          <p class="project-title">
+            
+            Wikipedia Viewer
+            
+          </p>
+        </a>
+        <a href="https://codepen.io/freeCodeCamp/full/KzXQgy" target="_blank" class="project project-tile">
+          <img class="project-image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png" alt="project">
+          <p class="project-title">
+            
+            Tic Tac Toe Game
+            
+          </p>
+        </a>
+      </div>
+
+      <a
+    href="https://codepen.io/FreeCodeCamp/"
+    class="button"
+    target="_blank"
+    >Show all<i class="fas fa-chevron-right"></i
+  ></a>
+    </section>    <!-- END PROJECTS SECTION -->
+    <!-- START CONTACT SECTION -->
+    <section id="contact" class="contact-section">
+      <div class="contact-section-header">
+        <h2>Let's work together...</h2>
+        <p>How do you take your coffee?</p>
+      </div>
+      <div class="contact-links">
+        <a href="https://facebook.com/freecodecamp" target="_blank" class="btn contact-details"><i class="fab fa-facebook-square"></i> Facebook</a>
+        <a id="profile-link" href="https://github.com/freecodecamp" target="_blank" class="btn contact-details"><i class="fab fa-github"></i> GitHub</a>
+        <a href="https://twitter.com/freecodecamp" target="_blank" class="btn contact-details"><i class="fab fa-twitter"></i> Twitter</a>
+        <a href="mailto:example@example.com" class="btn contact-details"><i class="fas fa-at"></i> Send a mail</a>
+        <a href="tel:555-555-5555" class="btn contact-details"><i class="fas fa-mobile-alt"></i> Call me</a>
+      </div>
```

Comment:
Your i-class icons in the contact-links class are not showing. I think you should fix this too

Reply:
- [OPTIONAL] In this case, I think it would be better to put it this way as to make the comment more polite and formal. Kindly check your social icons and fix the error as it is not showing in its present state.